packages/playground/src/components/common-breadcrumb/repository-dropdown.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { BreadcrumbDropdown, BreadcrumbDropdownOption } from './breadcrumb-dropdown'
+
+export type RepositoryDropdownOption = BreadcrumbDropdownOption<string>
+
+export interface RepositoryDropdownProps {
+  repositories: RepositoryDropdownOption[]
+  onSelectRepository: (selected: RepositoryDropdownOption) => void
+  selectedRepository?: RepositoryDropdownOption
+  isLast: boolean
+}
+
+export const RepositoryDropdown: React.FC<RepositoryDropdownProps> = ({
+  repositories,
+  onSelectRepository,
+  selectedRepository,
+  isLast
+}) => {
+  return (
+    <BreadcrumbDropdown
```

Comment:
is the only difference between all these dropdowns is the props, why do we need 4 separate components? cant we render the same component directly?